frontend/src/components/mine/Tailings/MineTailingsInfo.js

```diff
@@ -209,8 +219,9 @@ export class MineTailingsInfo extends Component {
           <hr style={{ borderTop: "2px solid #c4cdd5" }} />
```

Comment:
Can this <hr> be switched to the ant designs <Divider> ? 

We have a few throughout the app already, most take on the default styling, others are custom.

Reply:
Yes addressed.